lms/static/js/fixtures/commerce/checkout_receipt.html

```diff
@@ -0,0 +1,26 @@
+<div id="error-container" class="hidden">
+    <div id="error" class="wrapper-msg wrapper-msg-activate">
+        <div class=" msg msg-activate">
+            <span class="msg-icon icon fa fa-exclamation-triangle" aria-hidden="true"></span>
+
+            <div class="msg-content">
+                <h3 class="title">
+                    Error
+                </h3>
+                <div class="copy">
+                    <p>dummy error text</p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="container">
+    <section class="wrapper carousel">
+        <div id="receipt-container" class="pay-and-verify hidden" data-is-payment-complete='True'
+             data-platform-name='edx-platform' data-verified='True' data-username='user-1'>
+            <h2>${_("Loading Order Data...")}</h2>
```

Comment:
Nit: the fixture is intended to be rendered HTML, so it shouldn't include Python in it. It won't hurt as long as it is valid HTML, but it is a little confusing.